src/models/searchservice/response/HighlightedFields.ts

```diff
@@ -0,0 +1,41 @@
+import { HighlightedValue } from './HighlightedValue';
+
+/**
+ * A mapping of the values emphasized by the Answers API.
```

Comment:
nit: I would maybe add something like: The structure of HighlightedFields matches the structure of the corresponding rawData of a {@link Result}